src/admin.rs

```diff
@@ -439,3 +443,54 @@ where
 
     write_all_half(stream, res).await
 }
+
+/// Show currently connected clients
+async fn show_clients<T>(stream: &mut T) -> Result<(), Error>
+where
+    T: tokio::io::AsyncWrite + std::marker::Unpin,
+{
+    let columns = vec![
+        ("client_id", DataType::Text),
+        ("database", DataType::Text),
+        ("user", DataType::Text),
+        ("application_name", DataType::Text),
+        ("bytes_sent", DataType::Numeric),
+        ("bytes_received", DataType::Numeric),
+        ("transaction_count", DataType::Numeric),
+        ("query_count", DataType::Numeric),
+        ("error_count", DataType::Numeric),
+    ];
+
+    let new_map = {
+        let guard = CLIENT_STATES.read();
+        guard.clone()
+    };
+
+    let mut res = BytesMut::new();
+    res.put(row_description(&columns));
+
+    for (_, client) in new_map {
+        let row = vec![
+            format!("{:#08X}", client.process_id),
+            client.pool_name,
+            client.username,
+            client.application_name,
+            client.bytes_sent.to_string(),
+            client.bytes_received.to_string(),
+            client.transaction_count.to_string(),
+            client.query_count.to_string(),
+            client.error_count.to_string(),
```

Comment:
Being able to know which applications have the most active clients will be a very useful